### PR TITLE
DAM Block Index Admin (without Link to Dependency)

### PR DESCRIPTION
**Pull request by thomasdax98**
- Adds tab for displaying dependents of files in the DAM
- Adds DependencyList to `cms-admin` to easily implement a view for dependents or dependencies for other entities
- Adds `@EntityInfo` decorator to cms-api to provide additional information about an entity

Screenshot:

Bildschirmfoto 2024-03-07 um 16 07 05

---

Adapted from https://github.com/vivid-planet/comet/pull/953

- [ ] Add changeset (if necessary)

### CODE REVIEW

**Comment by thomasdax98** [comment]
I created a feature branch because I want to release it with all features (if possible). Also, I think this feature isn't "forgiving" enough yet and might break something. I want to thoroughly test it before realising it in a minor version.

File: packages/api/cms-api/src/index.ts [hunk 1]
```diff
@@ -69,6 +69,7 @@ export { defaultDamAcceptedMimetypes } from "./dam/common/mimeTypes/default-dam-
 export { DamConfig } from "./dam/dam.config";
 export { DAM_CONFIG, IMGPROXY_CONFIG } from "./dam/dam.constants";
 export { DamModule } from "./dam/dam.module";
+export { EntityInfo, EntityInfoServiceInterface } from "./dam/files/decorators/entity-info.decorator";
```
Comment (johnnyomair) [inline packages/api/cms-api/src/index.ts:72]: This file shouldn't be located in dam/files I suppose

File: packages/admin/cms-admin/src/dependencies/DependencyList.tsx [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+};
+
+type Dependent = {
+    rootGraphqlObjectType: string;
+    rootId: string;
+    rootColumnName: string;
+    jsonPath: string;
+    name: string | null;
+    secondaryInformation: string | null;
+};
```
Comment (johnnyomair) [inline packages/admin/cms-admin/src/dependencies/DependencyList.tsx:35]: Isn't this typed in the GraphQL schema?

File: packages/api/cms-api/src/index.ts (hunk 1 above)
Comment (thomasdax98) [reply packages/api/cms-api/src/index.ts]: Moved to /dependencies in 6553d558a39968a50339b8ce3fbf1d108286fd8f

**Review by johnnyomair — APPROVED** [review]

**Pull request merged**